cmd/bootstrap.go

```diff
@@ -21,59 +21,75 @@ package cmd
-	tequilapi_endpoints.AddRoutesForPilvytis(router, di.PilvytisAPI)
-	tequilapi_endpoints.AddRoutesForTerms(router)
-	tequilapi_endpoints.AddEntertainmentRoutes(router, entertainment.NewEstimator(
-		config.FlagPaymentPriceGiB.Value,
-		config.FlagPaymentPriceHour.Value,
-	))
-	tequilapi_endpoints.AddRoutesForValidator(router)
-	if err := tequilapi_endpoints.AddRoutesForSSE(router, di.StateKeeper, di.EventBus); err != nil {
-		return nil, err
-	}
-
-	if config.GetBool(config.FlagPProfEnable) {
-		tequilapi_endpoints.AddRoutesForPProf(router)
-	}
-
-	corsPolicy := tequilapi.NewMysteriumCorsPolicy()
-	return tequilapi.NewServer(listener, router, corsPolicy), nil
+	return tequilapi.NewServer(
+		listener,
+		[]func(engine *gin.Engine) error{
+			func(e *gin.Engine) error {
+				if err := tequilapi_endpoints.AddRoutesForSSE(e, di.StateKeeper, di.EventBus); err != nil {
+					return err
+				}
+				return nil
+			},
+			func(e *gin.Engine) error {
+				if config.GetBool(config.FlagPProfEnable) {
+					tequilapi_endpoints.AddRoutesForPProf(e)
+				}
+				return nil
+			},
+			func(e *gin.Engine) error {
+				e.GET("/healthcheck", tequilapi_endpoints.HealthCheckEndpointFactory(time.Now, os.Getpid).HealthCheck)
+				return nil
+			},
+			tequilapi_endpoints.AddRouteForStop(utils.SoftKiller(di.Shutdown)),
+			tequilapi_endpoints.AddRoutesForAuthentication(di.Authenticator, di.JWTAuthenticator),
+			tequilapi_endpoints.AddRoutesForIdentities(di.IdentityManager, di.IdentitySelector, di.IdentityRegistry, di.ConsumerBalanceTracker, di.AddressProvider, di.HermesChannelRepository, di.BCHelper, di.Transactor, di.BeneficiaryProvider, di.IdentityMover, di.PayoutAddressStorage),
+			tequilapi_endpoints.AddRoutesForConnection(di.ConnectionManager, di.StateKeeper, di.ProposalRepository, di.IdentityRegistry, di.EventBus, di.AddressProvider),
+			tequilapi_endpoints.AddRoutesForSessions(di.SessionStorage),
+			tequilapi_endpoints.AddRoutesForConnectionLocation(di.IPResolver, di.LocationResolver, di.LocationResolver),
+			tequilapi_endpoints.AddRoutesForProposals(di.ProposalRepository, di.PricingHelper, di.LocationResolver, di.FilterPresetStorage, di.NATProber),
+			tequilapi_endpoints.AddRoutesForService(di.ServicesManager, services.JSONParsersByType, di.ProposalRepository),
+			tequilapi_endpoints.AddRoutesForAccessPolicies(di.HTTPClient, config.GetString(config.FlagAccessPolicyAddress)),
+			tequilapi_endpoints.AddRoutesForNAT(di.StateKeeper, di.NATProber),
+			tequilapi_endpoints.AddRoutesForTransactor(di.IdentityRegistry, di.Transactor, di.HermesPromiseSettler, di.SettlementHistoryStorage, di.AddressProvider),
+			tequilapi_endpoints.AddRoutesForConfig,
+			tequilapi_endpoints.AddRoutesForMMN(di.MMN),
+			tequilapi_endpoints.AddRoutesForFeedback(di.Reporter),
+			tequilapi_endpoints.AddRoutesForConnectivityStatus(di.SessionConnectivityStatusStorage),
+			tequilapi_endpoints.AddRoutesForDocs, // TODO smt wrong
```

Comment:
```suggestion
			tequilapi_endpoints.AddRoutesForDocs,
```
Nothing wrong here anymore